lib/model.js

```diff
@@ -94,4 +94,17 @@ class Model {
   }
 }
 
+function checkEntities(entities) {
+  return [...entities]
```

Comment:
We don't need additional iterations, we already iterate entities in constructor, I'll fix and land.